guides/v2.1/cloud/docker/docker-database.md

```diff
@@ -0,0 +1,81 @@
+---
+group: cloud-guide
+title: Connect to the database
+functional_areas:
+  - Cloud
+  - Docker
+  - Configuration
+---
+
+There are two ways to connect to the database. Before you begin, you can find the database credentials in the `database` section of the `docker/config.php` file. The examples below use the following default credentials:
+
+> Filename: `config.php`
+
+```php
+<?php
+
+return [
+    'MAGENTO_CLOUD_RELATIONSHIPS' => base64_encode(json_encode([
+        'database' => [
+            [
+                'host' => 'db',
+                'path' => 'magento2',
+                'password' => 'magento2',
+                'username' => 'magento2',
+                'port' => '3306'
+            ],
+        ],
+    ...
+```
+{: .no-copy}
+
+#### To connect to the database using Docker commands:
```

Comment:
I'm not sure that this title and this "To connect to the database in local workstation:" is enough accurate and clear. @shiftedreality Could you please check it.

Reply:
Current version is fine

Reply:
I like that idea.